### DISCUSSION TITLE
GitHub Actions is currently disabled for your account

### DISCUSSION THREAD

**Discussion by bhanurp** [community/community#39888] [Actions]
### Select Topic Area

Bug

### Body

I am seeing this message on GitHub actions "GitHub Actions is currently disabled for your account" for an open-source project and jobs are getting queued without running for more than 15 hours. 

I have raised a support ticket but of no help as of now. Any workaround for this problem?

![image](https://user-images.githubusercontent.com/117445796/203666852-7f381b82-2de0-4ddd-a6f5-41b40133a6ee.png)

**Comment by ossanna16** [comment]
Hi there @bhanurp and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by richmiles** [comment]
I'm wondering if you have an update for this issue? I'm getting the same message and have reached out to support but haven't heard back.

**Reply by bhanurp** [reply]
@richmiles It took a very long time but Github support eventually reached out and enabled it for me.
